lib/handler.js

```diff
@@ -210,11 +210,33 @@ Handler.prototype.logout = async function logout () {
   this.redirect(redirectUrl)
 }
 
+Handler.prototype.setTokens = async function setTokens () {
```

Comment:
this could be a post with an endpoint `/auth/create`?

Would clean up extracting the tokens.